blockchain/fullblocktests/generate.go

```diff
@@ -2538,11 +2729,86 @@ func Generate(includeLargeReorg bool) (tests [][]TestInstance, err error) {
 	g.AssertTipNumRevocations(2)
 	rejected(blockchain.ErrRevocationsMismatch)
 
+	// Create block that has a revocation with more payees than expected.
+	//   ... -> brt1(24)
+	//                  \-> brt3(25)
+	g.SetTip("brt1")
+	g.NextBlock("brt3", outs[25], ticketOuts[25], func(b *wire.MsgBlock) {
+		for idx := 0; idx < len(b.STransactions); idx += 1 {
```

Comment:
Rather than increasing the test time by looping them, you can just use the knowledge that `brt1` is being built on, and it missed a vote, thus, this block will have the revocation at offset 10 (0-4 are the 5 votes and 5-9 are the 5 tickets requested by `ticketOuts`).  If you really want to be certain, it's a revocation, you can add an assertion for that specific tx.

The same goes for the remaining tests.